src/filters/weekDay/weekDay.test.ts

```diff
@@ -31,8 +31,8 @@ describe('Day Of The Week', () => {
             const result: ParsedMatchSchema[] = [
                 {
                     dateTime: mockWeekday(9),
-                    text: "hand in 's paper on mon",
-                    matched: 'tuesday',
+                    text: "hand 's paper on mon",
+                    matched: 'in tuesday',
```

Comment:
Had to do this due to the changes in matching words. I don't think there is an easy way around this problem - unless we specifically cater for the difference between months with and without an apostrophe and even then we would have phrases such as "in a month's time".

Reply:
Let's do this refactor on Friday?

Reply:
Aye